build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -1060,27 +1631,47 @@ new file mode 100644
 +  // Will schedule automatic updates, run in background.
 +  void Start();
 +
++  std::string GetAdBlockFiltersURL();
++  void SetAdBlockFiltersURL(const std::string url);
++  std::string GetMostRecentIndexedVersion();
++
++  // return date/time (in millis) of last successfully request
++  long GetLastOkUpdate();
++
++  // return date/time (in millis) of last request
++  long GetLastUpdate();
++
++  // Get/Set check interval (in days)
++  int GetAdBlockUpdateTimePeriod();
++  void SetAdBlockUpdateTimePeriod(int days);
++
 +  // To be called for an user-triggered update.
 +  // Will not result in an actual update if the last update was too recently triggered.
 +  bool OnDemandUpdate(Callback on_finished);
 +
++  static void RegisterPrefs(PrefRegistrySimple* registry);
++
 + private:
-+  void NotifyObservers(Event event);
++  void ReStart(int delay);
```

Comment:
Can we just call it `Start`? Since it is not doing any logic specific to restarting

Reply:
actually it is needed, because it is used in case the user changes the value in the settings

Reply:
There is no code specific to "restarting" vs "starting"

Reply:
maybe I don't understand, it's here

```
void AdBlockUpdaterService::SetAdBlockUpdateTimePeriod(int days) {
  pref_service_->SetInteger(kAdBlockFiltersCheckPeriod, days);
  ReStart(0);
}
```

Reply:
Inside the body of `ReStart` there is no logic like:
```
if (already_started) {
     // ...
}
```

This is what I mean

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/51b8c9b04f86cb1cb56e30abc152bfd539780bf9